index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/8b2919a184.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Mobile Portfolio</title>
+  </head>
+  <body>
+    <div class="main-container parent-div">
+      <div class="centered-div">
+        <nav class="nav-container">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
+          <div class="menu-bars">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </nav>
+        <main>
+          <div class="content-container">
+            <div class="heading-text">
+              <h2>I'm Marine</h2>
+              <h2>Glad to see you!</h2>
+            </div>
+            <div class="decription-text text">
+              <p>
+                I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+            </div>
+            <div class="message-text text">
+              <p>LET’S CONNECT</p>
```

Comment:
- [ ] Kindly wrap this `LET'S CONNECT` subtitle into the heading tag `hx (example: h2, h3, h4,...)`  because this way will be semantically correct and the screen readers and other technologies will thank you